src/osha/oira/content/browser/statistics.py

```diff
@@ -64,8 +59,8 @@ def render(self):
         session.execute("""DELETE FROM %s;""" % dbtable_surveys)
 
         def clean(value):
-            if isinstance(value, basestring):
-                return safe_unicode(value).strip().encode("utf-8")
+            if isinstance(value, str):
+                return safe_bytes(value.strip())
```

Comment:
Looking at the rest of the code, I think converting to bytes is wrong.
Anyway, I decided to keep the original behavior.